src/cosign/mod.rs

```diff
@@ -86,41 +86,53 @@ pub trait CosignCapabilities {
     ) -> Result<Vec<SignatureLayer>>;
 }
 
-/// Given a list of trusted `SignatureLayer`, find all the layers that satisfy
-/// the given constraints.
+/// Given a list of trusted `SignatureLayer`, find all the constraints that
+/// aren't satisfied by the layers.
+///
+/// If there's any unsatisfied constraints it means that the image failed
+/// verification.
+/// If there's no unsatisfied constraints it means that the image passed
+/// verification.
+///
+/// Returns a Result with either Ok() for passed verification or
+/// SigstoreError::SigstoreFilterConstraintsError{}, which contains a vec of
+/// references to unsatisfied constraints.
```

Comment:
```suggestion
/// Returns a `Result` with either `Ok()` for passed verification or
/// [`SigstoreFilterConstraintsError`](crate::errors::SigstoreFilterConstraintsError), which contains a vector of
/// references to unsatisfied constraints.
```

I improved the formatting, the link should work too. Double check by running `cargo doc --open`

Reply:
Oh, good point! Fixed that one (and another one), and added `cargo doc` as part of `make test` in https://github.com/sigstore/sigstore-rs/pull/42/commits/167114e21b7590569d878ed3126e4349ea5ea01b.